app/scripts/router.js

```diff
@@ -284,6 +284,8 @@ function (
         });
     },
 
+    showViewGenerator: showViewGenerator,
```

Comment:
@vladikoff : @ava14 was wondering today whether or not  you are going to do the merge, if he renames  the function?
thanks :)